guides/VERSIONING.md

```diff
@@ -14,3 +13,29 @@ Current version is 1.1.0
 * Minor release updates version to 1.2.0
 * Major release updates version to 2.0.0
 ```
+
+## Maintaining Old Versions
+
+When upgrading to a new major version, e.g. `1.*.*` to `2.*.*` it is digital publishing standard to cut a `v1` branch for the old version whilst the `main` branch will continue to contain the latest code for the new version (in this scenario v2).
```

Comment:
```suggestion
When upgrading to a new major version, e.g. `1.x` to `2.x` it is Digital Publishing standard to cut a `v1` branch for maintenance of the old major version. Then the `main` branch will continue to contain the latest code for the newest major version (in this scenario v2).
```

- I think it's more customary to write 1.x than 1.*
- some clarification in wording